### ISSUE TITLE
ktunnel should honor kubectl env var for setting the active config

### ISSUE THREAD

**Issue by fieldju** [omrikiei/ktunnel#15]
`export KUBECONFIG=${HOME}/.kube/dev-west.yaml`

When I use ktunnel I get the following error

```ERRO[0000] Failed getting kubernetes config: stat /Users/fieldju/.kube/config: no such file or directory ```

I would expect kubectl and ktunnel to use the same env var for choosing what config to use.

Found this: https://github.com/omrikiei/ktunnel/blob/53579ccf9fd56e3ca0efa012249e0b6fcab425df/pkg/k8s/common.go#L38

Is that a typo or intentional?

Right now for me to use kubectl and ktunnel I need to set 2 env vars

```bash
export KUBECONFIG=${HOME}/.kube/dev-west.yaml
export KUBE_CONFIG=$KUBECONFIG
```

**Comment by fieldju** [comment]
Found this: https://github.com/omrikiei/ktunnel/blob/53579ccf9fd56e3ca0efa012249e0b6fcab425df/pkg/k8s/common.go#L38

Is that a typo or intentional?

Right now for me to use kubectl and this I need to set 2 env vars

```bash
export KUBECONFIG=${HOME}/.kube/dev-west.yaml
export KUBE_CONFIG=$KUBECONFIG
```

**Comment by omrikiei** [comment]
it's a typo, it shall be fixed soon

**Comment by omrikiei** [comment]
released in v1.2.2
